examples/custom-indexer/rust/local-reader.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Sui Foundation.
```

Comment:
```suggestion
// Copyright (c) Mysten Labs, Inc.
```
I think the CI requires it this way.